libraries/AP_HAL_ChibiOS/hwdef/KakuteH7/hwdef.dat

```diff
@@ -0,0 +1,159 @@
+# hw definition file for processing by chibios_pins.py
+# for Holybro KakuteH7 hardware.
+
+# MCU class and specific type
+MCU STM32H7xx STM32H743xx
+
+# board ID for firmware load
+APJ_BOARD_ID 1048
+
+# crystal frequency, setup to use external oscillator
+OSCILLATOR_HZ 8000000
+
+FLASH_SIZE_KB 2048
+
+# ChibiOS system timer
+STM32_ST_USE_TIMER 12
+define CH_CFG_ST_RESOLUTION 16
+
+# leave 1 sectors free
+FLASH_RESERVE_START_KB 128
+
+# use last 2 pages for flash storage
+# H743 has 16 pages of 128k each
+define STORAGE_FLASH_PAGE 14
+define HAL_STORAGE_SIZE 32768
+
+# one I2C bus
+I2C_ORDER I2C1
+
+# order of UARTs (and USB)
+SERIAL_ORDER OTG1 USART1 USART2 USART3 UART4 UART7 USART6 OTG2
```

Comment:
For what it's worth I think its much more understandable to have a 1-1 mapping of UART -> SERIAL, so just leave SERIAL5 empty and switch UART6 and UART7

Reply:
done